# An example of how you can modify the PKGBUILD to support dev (non-versioned) builds.
--- PKGBUILD
+++ PKGBUILD
@@ -1,6 +1,6 @@
 # Maintainer: Phil Ruffwind <rf@rufflewind.com>
 pkgname=gitit
-pkgver=0.12.3
+pkgver=0.12.2.1.r26.gee9927f
 pkgrel=1
 pkgdesc="A wiki backed by a git, darcs, or mercurial filestore"
 arch=(i686 x86_64)
@@ -76,7 +77,7 @@ source=(haskell-ConfigFile-1.1.4.tar.gz::https://hackage.haskell.org/package/Con
         haskell-filestore-0.6.3.3.tar.gz::https://hackage.haskell.org/package/filestore-0.6.3.3/filestore-0.6.3.3.tar.gz
         haskell-ghc-paths-0.1.0.9-4.cabal::https://hackage.haskell.org/package/ghc-paths-0.1.0.9/revision/4.cabal
         haskell-ghc-paths-0.1.0.9.tar.gz::https://hackage.haskell.org/package/ghc-paths-0.1.0.9/ghc-paths-0.1.0.9.tar.gz
-        haskell-gitit-$pkgver.tar.gz::https://hackage.haskell.org/package/gitit-$pkgver/gitit-$pkgver.tar.gz
+        haskell-gitit-ee9927f8c22bc76353aa8589220b6aafc343847c.tar.gz::https://github.com/jgm/gitit/archive/ee9927f8c22bc76353aa8589220b6aafc343847c.tar.gz
         haskell-haddock-library-1.6.0.tar.gz::https://hackage.haskell.org/package/haddock-library-1.6.0/haddock-library-1.6.0.tar.gz
         haskell-happstack-server-7.5.1.1.tar.gz::https://hackage.haskell.org/package/happstack-server-7.5.1.1/happstack-server-7.5.1.1.tar.gz
         haskell-hashable-1.2.7.0-1.cabal::https://hackage.haskell.org/package/hashable-1.2.7.0/revision/1.cabal
@@ -392,11 +394,11 @@ EOF
         }
         tar -uf 00-index.tar "$name/$ver/$name.cabal"
     ) done
-    tar -xzf "haskell-gitit-$pkgver.tar.gz" "gitit-$pkgver/LICENSE"
+    tar -xzf haskell-gitit-ee9927f8c22bc76353aa8589220b6aafc343847c.tar.gz
 }
 
 build() {
-    cabal --config=.cabal/config install --enable-relocatable --force-reinstalls --datadir='$prefix/share/gitit' --docdir='$prefix/share/doc/$abi/$pkgid' --ghc-options=-rtsopts gitit
+    cabal --config=.cabal/config install --enable-relocatable --force-reinstalls --datadir='$prefix/share/gitit' --docdir='$prefix/share/doc/$abi/$pkgid' --ghc-options=-rtsopts ./gitit-ee9927f8c22bc76353aa8589220b6aafc343847c
 }
 
 package() {
@@ -404,5 +406,5 @@ package() {
     cp -PR .cabal-sandbox/share/gitit "$pkgdir/usr/share/"
     rm -fr "$pkgdir/usr/share/gitit/man"
     install -Dm755 .cabal-sandbox/bin/gitit "$pkgdir/usr/bin/gitit"
-    install -Dm644 "gitit-$pkgver/LICENSE" "$pkgdir/usr/share/licenses/gitit/LICENSE"
+    install -Dm644 gitit-ee9927f8c22bc76353aa8589220b6aafc343847c/LICENSE "$pkgdir/usr/share/licenses/gitit/LICENSE"
 }
